marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/security/getAuthoritiesTest.sjs

```diff
@@ -8,17 +8,18 @@ function invokeService() {
 }
 
 const response = invokeService();
-
+xdmp.log(response.authorities.toString());
+xdmp.log(response.roles.toString());
 [
     test.assertEqual("flow-developer", xdmp.getCurrentUser()),
     test.assertEqual(10, response.authorities.length),
     test.assertEqual("canWriteLoadData,canReadLoadData,canWriteFlows,canReadFlows," +
         "canWriteStepDefinitions,canReadStepDefinitions,canWriteMappings,canReadMappings,canWriteMatching,canReadMatching",
         response.authorities.toString()),
-    test.assertEqual(17, response.roles.length),
-    test.assertEqual("data-hub-operator,data-hub-entity-model-reader,data-hub-mapping-writer,data-hub-mapping-reader,"+
-    "data-hub-match-merge-writer,data-hub-load-data-writer,data-hub-module-writer,data-hub-load-data-reader,"+
-    "data-hub-job-reader,data-hub-flow-reader,data-hub-step-definition-reader,data-hub-match-merge-reader,data-hub-step-definition-writer,"+
-    "data-hub-flow-writer,data-hub-developer,data-hub-entity-model-writer,data-hub-module-reader",
-        response.roles.toString())
-]
+    test.assertEqual(19, response.roles.length),
```

Comment:
I'd change that to be an assertTrue(response.roles.length > 17, "with some message here explaining what the minimum number of expected roles is"). This assertion is otherwise going to break whenever roles are added.